docs/team/teojunda.md

```diff
@@ -5,42 +5,40 @@ title: "Axel's Project Portfolio Page"
 
 ### Project: TaToolkit Level 3
 
-TaToolkit - Level 3 is a desktop application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+TaToolkit is a desktop application used for teaching assistant to manage their students' personal details and other information related to their classroom. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 28 kLoC.
```

Comment:
small typo - teaching assistants *

Reply:
i see my ppp has the same issue, could I trouble you to help me change it :') thankss